config/deploy.rb

```diff
@@ -22,10 +25,4 @@
 # TASKS
```

Comment:
Looks good, thanks @alain-hoang. Only nitpick is to remove the whole `# Tasks` section now that it's empty.